### PR TITLE
Add Akka to integrations.yaml

### PR DESCRIPTION
**Pull request by sebastian-alfers**

### CODE REVIEW

**Comment by theletterf** [comment]
Hi @sebastian-alfers! Could you sign the CLA?

**Comment by sebastian-alfers** [comment]
Thank you for the reminder @theletterf, I am working on getting this signed.

File: data/ecosystem/integrations.yaml [hunk 1]
```diff
@@ -227,3 +227,8 @@
   docsUrl: https://docs.konghq.com/mesh/latest/guides/otel-metrics/
   components: [Go]
   oss: false
+- name: Akka
```
Comment (svrnm) [inline data/ecosystem/integrations.yaml:230]: Add `Akka` to line 1 such that cspell ignores it and is happy again

File: data/ecosystem/integrations.yaml [hunk 2]
```diff
@@ -227,3 +227,8 @@
...
+  url: https://akka.io/
+  docsUrl: https://developer.lightbend.com/docs/telemetry/current/plugins/opentelemetry/intro.html
```
Comment (svrnm) [inline data/ecosystem/integrations.yaml:232]: Reviewing this documentation it looks like that this only works with the Cinnamon Agent only?

**Review by svrnm — CHANGES_REQUESTED** [review]
An initial review, but to merge this, please make sure that you sign the CLA

Edit: Ah, @theletterf mentioned that already, apologies for duplicating that request

File: data/ecosystem/integrations.yaml (hunk 2 above)
Comment (sebastian-alfers) [reply data/ecosystem/integrations.yaml:232]: Correct. Cinnamon uses the OpenTelemetry client / api under the hood.

Comment (svrnm) [reply data/ecosystem/integrations.yaml:232]: From reading your documentation it sounds that (a) this is mandatory and (b) the agent is commercial?

**Comment by svrnm** [comment]
@open-telemetry/java-approvers please take a look

File: data/ecosystem/integrations.yaml (hunk 2 above)
Comment (sebastian-alfers) [reply data/ecosystem/integrations.yaml:232]: Yes, that seems right. If that is an issue, please let me know and I will close this pr.

Comment (svrnm) [reply data/ecosystem/integrations.yaml:232]: It's an unusual case what we have here. We have commercial integrations listed already, but we never had a situation where a commercial agent provides first party instrumentation for a library.

**Review by breedx-splk — COMMENTED** [review]
So, a couple of things. First, the OpenTelemetry Java Instrumentation itself does have some support for akka out of the box ([link](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/akka)). I'm not sure why we would want to link to a commercial product in that case.

Secondly, the Akka framework itself _is_ [open  source](https://github.com/akka/akka), so shouldn't this be `oss: true` instead of false?

**Comment by svrnm** [comment]
> So, a couple of things. First, the OpenTelemetry Java Instrumentation itself does have some support for akka out of the box ([link](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/akka)). I'm not sure why we would want to link to a commercial product in that case.

See [https://github.com/open-telemetry/opentelemetry.io/pull/4559#discussion_r1624631516]. We list commercial integrations already as long as they are native or first party. Our [integrations page](https://opentelemetry.io/ecosystem/integrations/) states the following:

> The following list contains a sample of libraries, services, and apps that have either integrated OpenTelemetry APIs and SDKs directly for native telemetry or provide a first-party plugin that fits into their own extensibility ecosystem.

The question is if this counts as _"first-party plugin that fits into their own extensibility ecosystem"_. Lightbend is the first-party behind akka, but to me it doesn't check the box for "extensibility ecosystem", since it requires a specific agent to accomplish that.

> Secondly, the Akka framework itself _is_ [open source](https://github.com/akka/akka), so shouldn't this be `oss: true` instead of false?

Akka is open source but the integration is not since it depends on the Cinnamon Agent. This of course adds to the confusion as people would expect something different when they read that Akka has native/first-party support for OpenTelemetry.

@sebastian-alfers LightBend Telemetry might fit into the vendors page, but based on the above we can not list it on the integrations page. If you disagree let me know.

**Comment by sebastian-alfers** [comment]
All good, you folks make the rules. Just wanted to mention, akka is not open source, we have established a source available BSL license.

Feel free to close this pr if you think it does not fit well, thanks again for taking a look.

**Comment by svrnm** [comment]
> All good, you folks make the rules. Just wanted to mention, akka is not open source, we have established a source available BSL license.
> 
> Feel free to close this pr if you think it does not fit well, thanks again for taking a look.

I am going to close it, but I encourage you to add LightBend to https://opentelemetry.io/ecosystem/vendors/

**Comment by sebastian-alfers** [comment]
> I am going to close it, but I encourage you to add LightBend to https://opentelemetry.io/ecosystem/vendors/

Thank you for the suggestion @svrnm. However, when I read the page it says:

> A non-exhaustive list of organizations offering solutions that consume OpenTelemetry natively via OTLP, such as observability backends and observability pipelines.

I am not sure if that fits, since we do not consume OTLP. We just export to receivers (like metrics and logs) using OTLP via http/grpc.

**Pull request closed without merging**